Makefile

```diff
@@ -58,9 +58,9 @@ endif
 
 test-with-kind:
 	@build/docker/build-image.sh intel/intel-fpga-admissionwebhook buildah
-	@$(PODMAN) tag localhost/intel/intel-fpga-admissionwebhook:devel docker.io/intel/intel-fpga-admissionwebhook:devel
+	@$(PODMAN) tag localhost/intel/intel-fpga-admissionwebhook:0.20.0 docker.io/intel/intel-fpga-admissionwebhook:0.20.0
```

Comment:
Perhaps `0.20.0` could be replaced with $(TAG) here?

Reply:
Oops, NVM. This should be fixed in the master branch first.

Reply:
@rojkov speaking of `test-with-kind`: I noticed there's a kind Github action. might be interesting to enable this for PRs.